data/remote_data_source/actors/src/test/java/com/sanaa/actors/ActorRemoteDataSourceImplTest.kt

```diff
@@ -0,0 +1,122 @@
+package com.sanaa.actors
+
+import com.example.env_config.service.LanguageProvider
+import com.sanaa.actors.dataSource.remote.ActorRemoteDataSource
+import com.sanaa.actors.dataSource.remote.dto.*
+import io.ktor.client.*
+import io.ktor.client.engine.mock.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import kotlinx.serialization.json.Json
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+import io.ktor.utils.io.ByteReadChannel
+import io.ktor.client.engine.mock.MockRequestHandleScope
+
+class ActorRemoteDataSourceImplTest {
+
+    private lateinit var client: HttpClient
+    private lateinit var dataSource: ActorRemoteDataSource
+    private lateinit var languageProvider: LanguageProvider
```

Comment:
You don't need to mock this object before each test, for faster tests, initialize the object directly 
```suggestion
    private var languageProvider: LanguageProvider = mockk()
```